support-frontend/assets/components/consentBanner/consentBanner.jsx

```diff
@@ -1,79 +1,83 @@
 // @flow
-import React from 'react';
-import type { Dispatch } from 'redux';
-import { connect } from 'react-redux';
-import type { ThirdPartyTrackingConsent } from '../../helpers/tracking/thirdPartyTrackingConsent';
+import React, { Component } from 'react';
+import { getTrackingConsent, writeTrackingConsentCookie, type ThirdPartyTrackingConsent } from '../../helpers/tracking/thirdPartyTrackingConsent';
 import {
   OptedIn,
   Unset,
 } from '../../helpers/tracking/thirdPartyTrackingConsent';
-import {
-  type Action,
-  setTrackingConsent,
-} from '../../helpers/page/commonActions';
 import Roundel from './roundel.svg';
 import Tick from './tick.svg';
 import Button from 'components/button/button';
 import Text from 'components/text/text';
 import styles from 'components/consentBanner/consentBanner.module.scss';
 import { isPostDeployUser } from 'helpers/user/user';
+import { ccpaEnabled } from 'helpers/tracking/ccpa';
 
-export type PropTypes = {
-  trackingConsent: ThirdPartyTrackingConsent,
-  onAccepted: Function,
+type StateTypes = {
+  showBanner: boolean,
 }
 
-function mapStateToProps(state) {
-  return {
-    trackingConsent: state.common.trackingConsent,
-  };
-}
+class ConsentBanner extends Component<{}, StateTypes> {
```

Comment:
You could use hooks to manage the state rather than a class as Tom F has done in [this PR](https://github.com/guardian/support-frontend/pull/2625)

Reply:
As discussed this `consentBanner.jsx` will be deprecated shortly so I'm going to avoid further updates  in this PR.